### PR TITLE
fix: changed fields to match model

### PR DESCRIPTION
**Pull request by TheMOHD3886**
## Description

Fixes date fields on the course cards. 

## Changes

- modifiedAt changed to dateUpdated
- createdAt changed to dateCreated

## Related Issues

None.

## Checklist

- [x] Code has been tested locally and passes all relevant tests.
- [x] Documentation has been updated to reflect the changes, if applicable.
- [x] Code follows the established coding style and guidelines of the project.
- [x] All new and existing tests related to the changes have passed.
- [x] Any necessary dependencies or new packages have been properly documented.
- [x] Pull request title and description are clear and descriptive.
- [x] Reviewers have been assigned to the pull request.
- [x] Any potential security implications have been considered and addressed.
- [x] Performance impact of the changes has been evaluated, if relevant.

## Screenshots (if applicable)

image

## If mobile/frontend pull request, what version of the backend is it stable, and running on? 

Branch: dev

Commit id: 6c101551026addb110d9c74e076deb407db92834

## Notes for Reviewers

None.

### CODE REVIEW

File: app/src/interfaces/Course.ts [hunk 1]
```diff
@@ -19,8 +19,8 @@ export interface Course {
   estimatedHours:    number;
   rating:            number;
   numOfSubscriptions:   number;
-  modifiedAt:        Date;
-  createdAt:         Date;
+  dateUpdated:       Date;
+  dateCreatde:       Date;
```
Comment (Jatewo) [inline app/src/interfaces/Course.ts:23]: Misspelled?

Comment (MortenMunk) [inline app/src/interfaces/Course.ts:23]: This is very bad standard... Please fix

**Review by MortenMunk — CHANGES_REQUESTED** [review]
We need to step up our game

_"this aggrestion will not stand, man.."_ - The Dude

**Review by Jatewo — CHANGES_REQUESTED** [review]
### Mediocre work
Fix the damn misspellings, kids

**Review by MortenMunk — APPROVED** [review]

**Review by Jatewo — APPROVED** [review]
### Ok, I give in

**Pull request merged**